coalib/testing/LocalBearTestHelper.py

```diff
@@ -57,8 +57,8 @@ def new_process_output(output, filename=None, file=None,
 
             bear_output_generator = bear.execute(*args, **kwargs)
 
-        assert bear_output_generator is not None, \
-            'Bear returned None on execution\n'
+        assert bear_output_generator is not None, ('Bear returned None on '
```

Comment:
per first comment, only replace `\` with `(` on this line, and then close the `)` on the next line.

or: avoid splitting a string if it can fit on one line.